src/runtime_src/core/edge/user/aie/aie.cpp

```diff
@@ -284,4 +284,23 @@ clear_bd(BD& bd)
 #endif
 }
 
+void
+Aie::
+reset(const std::shared_ptr<xrt_core::device>& device)
```

Comment:
My preference is to use "const xrt_core_device*" since the shared_ptr is not used at all and the device is in scope for the duration of the call.

Reply:
done